Arm/hand.hpp

```diff
@@ -0,0 +1,12 @@
+namespace sjsu::arm
+{
+class hand
+{
+  // The hand uses 5 'Actuonix PQ12-100-6-R' linear Actuators that use PWM
+  // signals to control their lengths.
+  //
+  // Need to confirm that that hand is
+  // controled by the same microcontrol as rest of arm, or is its own system.
```

Comment:
Actually the hand has its own MCU and talks over UART you will need to command it that way rather than directly using PWM.